clang/lib/CIR/Dialect/Transforms/TargetLowering/CIRLowerContext.cpp

```diff
@@ -19,12 +19,15 @@
 #include "clang/CIR/MissingFeatures.h"
 #include "llvm/Support/ErrorHandling.h"
 #include <cmath>
+#include <utility>
```

Comment:
Was this added as an IWYU-type thing?

Reply:
I believe this is added by clangd due to the use of `std::move`. Removed.